### PR TITLE
Add action route handler

### PR DESCRIPTION
**Pull request by mikkomustikkamaa**
Pull request contains implementation of action route handler to query single data provider with id

### CODE REVIEW

File: control-base/src/main/java/fi/nls/oskari/control/layer/GetDataProviderHandler.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+	private DataProviderService service = ServiceFactory.getDataProviderService();
+
+	@Override
+	public void handleAction(ActionParameters params) throws ActionException {
+		try {
+			final int id = Integer.valueOf(params.getHttpParam(PARAM_ID));
+			DataProvider d = service.find(id);
+			final JSONObject result = new JSONObject();
+			JSONHelper.putValue(result, "dataprovider", d.getAsJSON());
```
Comment (ZakarFin) [inline control-base/src/main/java/fi/nls/oskari/control/layer/GetDataProviderHandler.java]: I dont think the response needs to we wrapped in an additional JSON object. Just return the getAsJSON() as is? Also consider renaming this to DataProvider extending RestActionHandler so we could combine the save/delete to this as well and get rid of https://github.com/oskariorg/oskari-server/blob/master/control-base/src/main/java/fi/nls/oskari/control/layer/SaveOrganizationHandler.java etc

File: control-base/src/main/java/fi/nls/oskari/control/layer/GetDataProviderHandler.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+public class GetDataProviderHandler extends ActionHandler {
+
+	private static final String PARAM_ID = "id";
+
+	private DataProviderService service = ServiceFactory.getDataProviderService();
+
+	@Override
+	public void handleAction(ActionParameters params) throws ActionException {
+		try {
+			final int id = Integer.valueOf(params.getHttpParam(PARAM_ID));
```
Comment (ZakarFin) [inline control-base/src/main/java/fi/nls/oskari/control/layer/GetDataProviderHandler.java]: There's probably some params.getRequiredParamInt() that you could use here

File: control-base/src/main/java/fi/nls/oskari/control/layer/GetDataProviderHandler.java (hunk 1 above)
Comment (mikkomustikkamaa) [reply control-base/src/main/java/fi/nls/oskari/control/layer/GetDataProviderHandler.java]: Fixed but left mentioned combination of save and delete to future

File: control-base/src/main/java/fi/nls/oskari/control/layer/GetDataProviderHandler.java (hunk 2 above)
Comment (mikkomustikkamaa) [reply control-base/src/main/java/fi/nls/oskari/control/layer/GetDataProviderHandler.java]: Fixed. This is nice and makes code cleaner in the handler

**Pull request merged**